src/pages/workspace/tags/WorkspaceTagsPage.tsx

```diff
@@ -109,12 +111,14 @@ function WorkspaceTagsPage({route}: WorkspaceTagsPageProps) {
     });
 
     const getPendingAction = (policyTagList: PolicyTagList): PendingAction | undefined => {
-        if (!policyTagList) {
+        if (!policyTagList ?? !policyTagList.tags) {
             return undefined;
         }
```

Comment:
Why is this change required? I'm getting an ts error on this line `Right operand of ?? is unreachable because the left operand is never nullish.ts(2869)`